src/geode/model/helpers/convert_brep_section.cpp

```diff
@@ -188,51 +195,52 @@ namespace
                 line_slice1, surface );
             add_extruded_line_surface_boundary_relationship(
                 section_line, surface );
+
             auto surface_builder =
                 brep_builder_.surface_mesh_builder( surface.id() );
-            const auto prev_slide0_pointid =
-                line_slice0.mesh().edge_vertex( { 0, 0 } );
-            auto prev_slice0_pt = surface_builder->create_point(
-                line_slice0.mesh().point( prev_slide0_pointid ) );
-            brep_builder_.set_unique_vertex(
-                { surface.component_id(), prev_slice0_pt },
-                brep_.unique_vertex(
-                    { line_slice0.component_id(), prev_slide0_pointid } ) );
-            const auto prev_slide1_pointid =
-                line_slice1.mesh().edge_vertex( { 0, 0 } );
-            auto prev_slice1_pt = surface_builder->create_point(
-                line_slice1.mesh().point( prev_slide1_pointid ) );
-            brep_builder_.set_unique_vertex(
-                { surface.component_id(), prev_slice1_pt },
-                brep_.unique_vertex(
-                    { line_slice1.component_id(), prev_slide1_pointid } ) );
             for( const auto edge_id :
                 geode::Range{ line_slice0.mesh().nb_edges() } )
             {
-                const auto slide0_pointid =
-                    line_slice0.mesh().edge_vertex( { edge_id, 1 } );
-                const auto slide0_pt = surface_builder->create_point(
-                    line_slice0.mesh().point( slide0_pointid ) );
-                brep_builder_.set_unique_vertex(
-                    { surface.component_id(), slide0_pt },
-                    brep_.unique_vertex(
-                        { line_slice0.component_id(), slide0_pointid } ) );
-                const auto slide1_pointid =
-                    line_slice1.mesh().edge_vertex( { edge_id, 1 } );
-                const auto slice1_pt = surface_builder->create_point(
-                    line_slice1.mesh().point( slide1_pointid ) );
-                brep_builder_.set_unique_vertex(
-                    { surface.component_id(), slice1_pt },
-                    brep_.unique_vertex(
-                        { line_slice1.component_id(), slide1_pointid } ) );
-                surface_builder->create_polygon(
-                    { prev_slice0_pt, prev_slice1_pt, slice1_pt, slide0_pt } );
-                prev_slice0_pt = slide0_pt;
-                prev_slice1_pt = slice1_pt;
+                std::array< geode::index_t, 4 > pointids;
+
```

Comment:
avoid line space. either it has a meaning and then create a block function or remove it